lib/consumer/amqp_data_consumer.ex

```diff
@@ -52,8 +52,8 @@ defmodule Mississippi.Consumer.AMQPDataConsumer do
 
   # This is a Message Tracker deactivating itself normally, just remove its monitor.
   # In case a messageTracker crashes, we want to crash too, so that messages are requeued.
```

Comment:
Let's update the comment here so that it's clear why `{:shutdown, :requested}` is a normal reason

Reply:
I've slightly refactored the crashing mechanism to avoid overly cluttering that comment